python/_tskitmodule.c

```diff
@@ -736,6 +736,31 @@ make_file(PyObject *fileobj, const char *mode)
     return ret;
 }
 
+static int
+uint32_converter(PyObject *py_obj, uint32_t *uint_out)
+{
+    int64_t temp_long;
+    int ret = 0;
+
+    if (!PyArg_Parse(py_obj, "l", &temp_long)) {
```

Comment:
Probably best to use [this function](https://docs.python.org/3/c-api/long.html#c.PyLong_AsLongLongAndOverflow) if we want to be absolutely 100% sure.

Reply:
Hmm, looks like we're already catching overflows properly anyway, so maybe no need to bother with that. We can't assume a long is 64 bit though, we need a ``long long`` for that.

Reply:
I tried `PyLong_AsLongLongAndOverflow` but it doesn't like the `np.uint32` objects that `add_row` currently recieves.

Reply:
Dammit. PyArg_Parse is good anyway, just use a long long. That's probably the issue on windows, as a long is 32 bits there.